model_definitions/03c9a01f-bd46-4e7c-9a60-4282039094e6/model_modules/training.py

```diff
@@ -12,7 +12,9 @@
 def train(data_conf, model_conf, **kwargs):
     hyperparams = model_conf["hyperParameters"]
 
-    create_context(host=data_conf["host"], username=os.environ['TD_USERNAME'], password=os.environ['TD_PASSWORD'])
+    create_context(host=os.environ["AOA_CONN_HOST"],
```

Comment:
Something to think about.. Should we provide a nicer/more concise way to access these variables? I can't think of anything that would make a real difference and it would only be abstracting unnecessarily. The only thing is maybe use shorter env var names?

Reply:
I think is fine for now, the var names are self explanatory and not so long. But I'll give it a thought